src/SprykerEco/Zed/Computop/Business/Payment/Handler/Saver/Complete/PayPalExpressCompleteResponseSaver.php

```diff
@@ -0,0 +1,185 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Computop\Business\Payment\Handler\Saver\Complete;
```

Comment:
I would suggest that we move it from the Handler namespace

Reply:
It will break directories structure. 
Zed/Computop/Business/Payment/Handler/Saver/ - is already exists with old classes